<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogListeRechercheSouhaits</class>
 <widget class="QDialog" name="DialogListeRechercheSouhaits">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_barreMenu" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="label_ListeSouhaits">
     <property name="font">
      <font>
       <family>Liberation Sans</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Caract√©ristiques du bien</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_EtiquettePrix">
       <property name="font">
        <font>
         <family>Liberation Sans</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Prix</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_Prix">
       <property name="font">
        <font>
         <family>Liberation Sans</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>prix</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_EtiquetteSurfHab">
       <property name="font">
        <font>
         <family>Liberation Sans</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Surface habitable</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_SurfHab">
       <property name="font">
        <font>
         <family>Liberation Sans</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>surfHab</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_EtiquetteSurfJard">
       <property name="font">
        <font>
         <family>Liberation Sans</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Surface jardin</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_SurfJard">
       <property name="font">
        <font>
         <family>Liberation Sans</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>surfJard</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_EtiquetteVille">
       <property name="font">
        <font>
         <family>Liberation Sans</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ville</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_Ville">
       <property name="font">
        <font>
         <family>Liberation Sans</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ville</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget_ListeSouhaits"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
